exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/PcapBatchReader.java

```diff
@@ -190,6 +199,8 @@ public boolean next() {
 
   @Override
   public void close() {
+    logger.warn("Unclosed sessions remaining in PCAP");
```

Comment:
Could you please explain this warning? How it can be avoided?

Reply:
This warning happens in the event of an incomplete or corrupt PCAP file.

Reply:
I added an if statement here.